R/show.R

```diff
@@ -285,7 +285,13 @@ showMultitable <- function (x) {
     # eg remove selected_array()
     out <- c(out, "Column variables:",
              vapply(x@body$template, function (expr) {
-                 paste0("  ", formatExpression(expr$query[[1]]))
+                 if (names(expr$query[[1]]) == "each") {
+                     # if the first element of the query is each, then this is
+                     # an MR so take the second argument instead.
```

Comment:
Actually, it's any array variable, not just MR. Though maybe you can't put a categorical array in a multitable?

And anyway, looking at the test output again, I'm not sure there's any value in showing `as_selected(allpets)` instead of just `allpets`.  So if we're going to special-case in here, perhaps we should get that right. (Recognizing that this is scope creep here)